purchase_security/security/security.xml

```diff
@@ -47,5 +65,13 @@
             >['|',('order_id.user_id','=',user.id),('order_id.user_id','=',False)]</field>
             <field name="groups" eval="[(4,ref('group_purchase_own_orders'))]" />
         </record>
+        <record model="ir.rule" id="purchase_order_user_group_purchases_rule">
+            <field name="name">View purchase order lines (purchase responsible)</field>
+            <field name="model_id" ref="purchase.model_purchase_order_line" />
+            <field
+                name="domain_force"
+            >['|',(user.id, 'in', 'order_id.team_id.user_ids'),('order_id.team_id', '=',False)]</field>
```

Comment:
@stefan-tecnativa please at least make the minimum functional tests before asking for reviews. This is wrong